### PR TITLE
[Magic Transit] Updated MT onboarding

### PR DESCRIPTION
**Pull request by dcpena**
Implemented feedback on MT onboarding process. Addresses [PCX-1289](https://jira.cfops.it/browse/PCX-1289)

### CODE REVIEW

File: products/magic-transit/src/content/use-magic-transit-with-cni.md [hunk 1]
```diff
@@ -9,18 +9,26 @@ order: 4
...
-2. You order the cross-connects that you want to use with CNI. You can use any of the following:
+2. Cloudflare generates Letters of Authorization (LOAs) for your CNI cross-connects and sends them to your organization.
+
+3. Order the cross-connects that you want to use with CNI. You can use any of the following:
     * **Private network interconnects (PNI)** are available at any of our [private peering facilities](https://www.peeringdb.com/net/4224).
     * **Virtual private network interconnects (vPNI)** allow you to easily connect with Cloudflare at any of our interconnection platform locations.
     * **Internet exchange point (IXP)** interconnects allow you to establish a link with Cloudflare at any of the [more than 200 IXPs](https://bgp.he.net/AS13335#_ix) in which we participate.
 
-3. You send Cloudflare confirmation when the cross-connects are in place.
+4. You send Cloudflare confirmation when the cross-connects are in place.
```
Comment (nayaabahsan) [inline products/magic-transit/src/content/use-magic-transit-with-cni.md]: It sounds a bit awkward here, I think you can directly say "Send Cloudflare confirmation..." like in step 1

File: products/magic-transit/src/content/use-magic-transit-with-cni.md [hunk 2]
```diff
@@ -9,18 +9,26 @@ order: 4
...
+
+5. Cloudflare supplies customers with the GRE IPs and BGP Peering info, which is completed after GRE tunnels in CNI links are onboarded.
+
```
Comment (nayaabahsan) [inline products/magic-transit/src/content/use-magic-transit-with-cni.md:24]: is passive voice necessary here? for example, could be "...completed after Cloudflare onboards GRE tunnels in CNI links." or "...completed after onboarding GRE tunnels in CNI links." or "Cloudflare provides the GRE IPs and BGP Peering info after onboarding GRE tunnels in CNI links."
(disregard if none of this applies, I'm not familiar with the technicalities)

File: products/magic-transit/src/content/use-magic-transit-with-cni.md [hunk 3]
```diff
@@ -9,18 +9,26 @@ order: 4
...
+6. The customer and Cloudflare work to establish the BGP session for the PNI on both sides. A BGP call is needed, and the customer provides a ~2hr maintenance window.
+
```
Comment (nayaabahsan) [inline products/magic-transit/src/content/use-magic-transit-with-cni.md:26]: Similar to comment on step 4, I think it's best not to switch between addressing the customer as "you" and "the customer" - we generally say "you." So here, "Work with Cloudflare to..."

and in step 7, "Configure the GRE..."
Again, maybe that doesn't totally work in this context, but commenting if it does.

File: products/magic-transit/src/content/use-magic-transit-with-cni.md (hunk 2 above)
Comment (dcpena) [reply products/magic-transit/src/content/use-magic-transit-with-cni.md:24]: I agree; it's unnecessary. Fixed!

File: products/magic-transit/src/content/use-magic-transit-with-cni.md (hunk 3 above)
Comment (dcpena) [reply products/magic-transit/src/content/use-magic-transit-with-cni.md:26]: I like "Work with Cloudflare...". Fixed!

File: products/magic-transit/src/content/use-magic-transit-with-cni.md (hunk 1 above)
Comment (dcpena) [reply products/magic-transit/src/content/use-magic-transit-with-cni.md]: Fixed

**Review by nayaabahsan — APPROVED** [review]
Approving for content!

**Pull request merged**